src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ReviewEvidenceNoteViewModelMap.cs

```diff
@@ -25,6 +25,8 @@ public ReviewEvidenceNoteViewModel Map(ViewEvidenceNoteMapTransfer source)
                 ViewEvidenceNoteViewModel = mapper.Map<ViewEvidenceNoteViewModel>(source)
             };
 
+            model.ViewEvidenceNoteViewModel.DisplayH2Title = true;
```

Comment:
Update unit test to check that the DisplayH2Title by the Map function

Reply:
added unit test to cover this property